### PR TITLE
Create Index for Terms And Privacy Pages

### PR DESCRIPTION
**Pull request by darksbond**

### CODE REVIEW

File: packages/client/components/pages/privacy/Privacy.tsx [hunk 1]
```diff
@@ -1,13 +1,50 @@
...
+      <List
+        sx={{
+          width: '100%',
+          maxWidth: '600px',
+          mx: 'auto',
+          mt: '2rem',
+          mb: '2rem'
+        }}
+      >
+        <StyledLink href='/privacy#concent'>Concent</StyledLink>
```
Comment (paradus-hex) [inline packages/client/components/pages/privacy/Privacy.tsx:21]: Kindly fix the spelling of Concent. It should be Consent.

File: packages/client/components/pages/privacy/Privacy.tsx [hunk 2]
```diff
@@ -1,13 +1,50 @@
...
+        <StyledLink href='/privacy#information-we-collect'>
```
Comment (paradus-hex) [inline packages/client/components/pages/privacy/Privacy.tsx:22]: Do not write the href link this. Write it like this instead '#information-we-collect', no need to add the name of the current page. Fix this for all such instances on the privacy and terms page.

File: packages/client/components/pages/terms/Terms.tsx [hunk 3]
```diff
@@ -1,13 +1,41 @@
...
+      <List
+        sx={{
+          width: '100%',
+          maxWidth: '600px',
+          mx: 'auto',
+          mt: '2rem',
+          mb: '2rem'
+        }}
+      >
+        <StyledLink href='/terms#cookies'>Cookies</StyledLink>
```
Comment (paradus-hex) [inline packages/client/components/pages/terms/Terms.tsx:21]: The index titles should all be serially numbered from 1 to 10.

**Review by Devorein — APPROVED** [review]

File: packages/client/components/pages/terms/Terms.tsx (hunk 3 above)
Comment (darksbond) [reply packages/client/components/pages/terms/Terms.tsx]: solved

**Comment by darksbond** [comment]
Done

**Pull request merged**